sdk/src/main/java/io/opentelemetry/sdk/trace/ReadableSpan.java

```diff
@@ -38,13 +48,106 @@
    * Span#updateName(String)} so this value cannot be cached.
    *
    * @return the name of the {@code Span}.
+   * @since 0.1.0
    */
   String getName();
 
   /**
-   * Returns the proto representation of the collected data for this particular {@code Span}.
+   * Returns the value of System.nanoTime() when the span was started.
    *
-   * @return the proto representation of the collected data for this particular {@code Span}.
+   * @return Long value representing the System.nanoTime().
+   * @since 0.1.0
+   */
+  long getStartNanoTime();
```

Comment:
Not sure I understand why not having a toSpanData instead of all getters, I know that we discussed this as an alternative but I thought we agreed to go with toSpanData.